garrysmod/gamemodes/sandbox/gamemode/commands.lua

```diff
@@ -855,6 +855,19 @@ function Spawn_Weapon( ply, wepname, tr )
 
 	gamemode.Call( "PlayerSpawnedSWEP", ply, entity )
 
+	local PrintName = swep.PrintName
+
+    	-- We haven't got a nice name :/

```

Comment:
language.GetPhrase on the client

Reply:
Yeah...? But isn't `commands.lua` a serverside-only file?

Reply:
I had no idea the undo module just networked the string straight to the client, and now I wish I didn't.